### ISSUE TITLE
On macOS, a Casual value of 0 appears the same as a value of 1

### ISSUE THREAD

**Issue by rudedogg** [arrowtype/recursive#442]
It seems that using a `Casual` value of 0 looks identical to using a value of 1.

However, a low, but non-zero value (ie: 0.01) works as expected. I've seen this behavior/bug in both an in-development app and Sketch. Both behave the same.

Maybe it's a macOS bug? I don't see the problem in Safari when browsing https://www.recursive.design

**Expected behavior**
A Casual value of 0 should be very straight, and not curvy. Like on https://www.recursive.design

**Screenshots**
Demo app:
![image](https://user-images.githubusercontent.com/2127629/109377724-56ae1000-788a-11eb-9ecd-12697514b6a7.png)
Sketch 63.1:
![image](https://user-images.githubusercontent.com/2127629/109377610-99bbb380-7889-11eb-9321-e2775952bf61.png)
Barely non-zero Casual value:
![image](https://user-images.githubusercontent.com/2127629/109377639-ba840900-7889-11eb-915d-219488643421.png)

**Environment (please complete the following information):**
- OS: `macOS Big Sur Version 11.2.2`. 
- Browser: N/A
- Fonts: v 1.074 (installed `Recursive_VF_1.074.ttf`)

**Additional context**
Sketch:  Version 63.1

**Comment by arrowtype** [comment]
Thanks for taking time to file this issue, @rudedogg!

There is a similar issue at https://github.com/arrowtype/recursive/issues/317, though it is closed because A) it is outside of my control, and B) I thought it might be fixed by macOS 11 (though I wasn’t able to test this very much). I also previously filed an [Apple Feedback/Radar](https://feedbackassistant.apple.com/feedback/7587364), but they claimed in there that this was a bug in Sketch.

> I've seen this behavior/bug in both an in-development app

Are you able to say more about this? E.g. is this an app you are development? Do you know if it is using basic system APIs? Full disclosure: I know very little about Mac development, but if this were possible to isolate to a specific API, it would probably help Apple to fix it. If it can’t be, maybe it really is a Sketch bug.

**Comment by lkhphuc** [comment]
I have a somewhat similar problem in Firefox with MacOS.
Just by setting default font as "Recursive" in Firefox broke both Mono and Casual style, like this:
![Screenshot 2021-06-05 at 12 54 36](https://user-images.githubusercontent.com/12573521/120890851-30241180-c5fd-11eb-9c53-9043e2934350.png)

My workaround for the moment is using `Stylus` plugin and inject this `css` on every website:
```
body, span, div, a, p, select, h1, h2, h3, h4, h5, li {
    font-variation-settings: "CASL" 0.001, "MONO" 0.001, "CRSV" 0.5 !important;
}

textarea, input, button, h6 {
    font-variation-settings: "CASL" 1, "MONO" 0.001, "CRSV" 0.5 !important ;
}
```

**Comment by rudedogg** [comment]
> Thanks for taking time to file this issue, @rudedogg!
> 
> There is a similar issue at #317, though it is closed because A) it is outside of my control, and B) I thought it might be fixed by macOS 11 (though I wasn’t able to test this very much). I also previously filed an [Apple Feedback/Radar](https://feedbackassistant.apple.com/feedback/7587364), but they claimed in there that this was a bug in Sketch.
> 
> > I've seen this behavior/bug in both an in-development app
> 


@arrowtype I want to apologize for taking so long to get back to you!

This issue looks fixed to me in the macOS 12.0 beta: 

![Screen Shot 2021-09-12 at 9 45 17 PM](https://user-images.githubusercontent.com/2127629/133021172-01a81045-5e61-4840-b427-7f04a31a04e8.png)


> Are you able to say more about this? E.g. is this an app you are development? Do you know if it is using basic system APIs? 

Yes, it is an app under development, but I am using `NSFont`/`NSFontDescriptor`. Those APIs would be the most basic/common way on macOS to use a custom font in a native app.

@lkhphuc Hopefully Firefox is using this API, and your issue will be fixed on the next macOS release too.

**Comment by arrowtype** [comment]
That's good news; thanks for the update! I'll close this, since a fix seems to be on the way.
